examples/src/main/scala/org/apache/spark/examples/BroadcastTest.scala

```diff
@@ -42,7 +42,7 @@ object BroadcastTest {
     val arr1 = (0 until num).toArray
 
     for (i <- 0 until 3) {
-      println("Iteration " + i)
+      println(s"Iteration ${i}")
```

Comment:
Beyond the unnecessary { } that @srowen has already mentioned, this isn't really a style improvement. `"a string " + anotherString` is arguably at least as good stylistically as using string interpolation for such simple concatenations of a string reference to the end of a string literal. It's only when there are multiple concatenations and/or multiple string references that interpolation is clearly the better way.

Reply:
@markhamstra Thank you for valueable suggestion, I am addressed and did new commit.